aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -338,25 +334,73 @@ module post_mint_reveal_nft::minting {
         };
     }
 
-    /// Add user addresses to the whitelist.
+    /// Set up different stages of the whitelist config.
+    /// Most of the times, NFT projects will only have one-stage of whitelist.
+    /// In that case, the `curr_stage` being passed in should be 0.
+    /// If there are multiple stages of whitelist, the `curr_stage` should be the
+    /// current stage of the whitelist (starting from 0).
+    /// For example, if there are 3 stages of whitelist,
+    /// we need to call this function three times, with curr_stage being 0, 1, 2 respectively.
+    public entry fun set_up_whitelist_stage(
+        admin: &signer,
+        curr_whitelist_stage_start_time: u64,
+        curr_whitelist_stage_end_time: u64,
+        curr_whitelist_stage_price: u64,
+        curr_stage: u64,
+    ) acquires WhitelistMintConfig, NFTMintConfig {
+        let nft_mint_config = borrow_global_mut<NFTMintConfig>(@post_mint_reveal_nft);
+        assert!(signer::address_of(admin) == nft_mint_config.admin, error::permission_denied(ENOT_AUTHORIZED));
+
+        if (exists<WhitelistMintConfig>(nft_mint_config.resource_address)) {
+            let next_stage_index = vector::length(&borrow_global<WhitelistMintConfig>(nft_mint_config.resource_address).whitelist_configs);
```

Comment:
Is it possible to make a WhitelistMintConfig but with zero items in `whitelist_configs`? If so, this will skip the zeroth index. Just double checking.